src/DynamoMLDataPipeline/DynamoMLDataPipeline.cs

```diff
@@ -49,34 +49,53 @@ internal string ProductionCollectionID
             }
         }
 
-        internal static IOAuth2AccessTokenProvider AuthTokenProvider { get; set; }
+        internal DynamoModel DynamoModel { get; set; }
+
+        internal static IDSDKManager AuthProvider { get; set; }
```

Comment:
I'd think hard about making this static...

Reply:
Removed static modifier.